src/include/firewall/uifunctions.rb

```diff
@@ -135,21 +144,33 @@ def SaveAndRestart
     def StartNow
       UI.OpenDialog(Label(_("Starting firewall...")))
       SuSEFirewall.SetStartService(true)
-      ret = SuSEFirewall.StartServices
+      success = SuSEFirewall.StartServices
+      Popup.Error(
+        _(
+          "The firewall could not be started.\n" +
+          "Please verify your system and try again."
+        )
+      ) unless success
       UI.CloseDialog
 
-      ret
+      success
     end
 
     # Function stops Firewall services and sets firewall
     # to be stopped after exiting YaST
     def StopNow
       UI.OpenDialog(Label(_("Stopping firewall...")))
       SuSEFirewall.SetStartService(false)
-      ret = SuSEFirewall.StopServices
+      success = SuSEFirewall.StopServices
+      Popup.Error(
+        _(
+          "The firewall could not be stopped.\n" +
+          "Please verify your system and try again."
+        )
```

Comment:
Composing sentences is very tricky if you have to do it across languages, as illustrated with a quick IRC experiment:
```
<a> b:  # TRANSLATORS: action which failed, used later
<a> b:  _("stopped")
<a> b: now, without looking at the PR, write the correct Czech translation here
<b> a: "zastavil"?
<b> or "zastaveni"
<a> b: wrong, thank you for proving my point :)
<a> the correct string is Zastavení
<b> so second one is correct :)
<a> no, you got the capitalization wrong
```

If we want to factor out parts of translatable strings, I would do it at sentence boundaries to minimize problems.